src/engraving/layout/v0/chordlayout.cpp

```diff
@@ -2292,10 +2292,11 @@ void ChordLayout::layoutChords3(const MStyle& style, const std::vector<Chord*>&
     int prevSubtype = 0;
     int prevLine = std::numeric_limits<int>::min();
 
+    bool isTab = staff->isTabStaff(tick);
     for (int i = nNotes - 1; i >= 0; --i) {
         Note* note     = notes[i];
         Accidental* ac = note->accidental();
-        if (ac && ac->subtype() == prevSubtype && note->line() == prevLine) {
+        if (ac && (isTab || (ac->subtype() == prevSubtype && note->line() == prevLine))) { // TODO: probably this function shouldn't be called at all for tab staves
```

Comment:
I think you're probably right.  The main doubt for me is the one line to layout stemSlash a bit further down.  But it seems that happens other places too and I'm not sure there anything special that should need to be done during this pass.